### ISSUE TITLE
If the rating dialog is displayed successfully on iOS, the result will not be returned.

### ISSUE THREAD

**Issue by randnetdd** [dpa99c/cordova-launch-review#24]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](README#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [x] I have reproduced the issue using the example projector provided the necessary information to reproduce the issue.

- [x] I have read the documentation thoroughly and it does not help solve my issue.

- [ ] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**

If the rating dialog is displayed successfully on iOS, the result will not be returned.

**Expected behavior:**

result returns show if rating dialog is displayed successfully on iOS

**Steps to reproduce:**

I haven't written any special code

**Screenshots**

![image0 2](https://user-images.githubusercontent.com/33739615/95716621-b4103e80-0ca6-11eb-9e4b-bae6d15182fa.png)
![unnamed 1](https://user-images.githubusercontent.com/33739615/95716539-97740680-0ca6-11eb-80be-fe82f3344050.png)

**Environment information**

- Cordova CLI version 
	- 10.0.0
- Cordova platform version
	- ios@6.1.1
- Plugins & versions installed in project (including this plugin)
     cordova-launch-review 4.0.0 "Launch Review"
     cordova-plugin-device 2.0.3 "Device"
     cordova-plugin-whitelist 1.3.4 "Whitelist"
        
_Runtime issue_
- Device details
    - iPhone X
- OS details
    - iOS 13.3.1	
	
_iOS build issue:_
- Node JS version
    - 10.16.0
- XCode version
    - 12

**Related code:**
```
LaunchReview.rating(result => {
    console.log('result: ' + result);
    if (cordova.platformId === 'android') {
        console.log('Rating dialog displayed');
    } else if (cordova.platformId === 'ios') {
        if (result === 'requested') {
            console.log('Requested display of rating dialog');
            this.ratingTimerId = setTimeout(() => {
                console.warn('Rating dialog was not shown (after ' + 5000 + 'ms)');
            }, 5000);
        } else if (result === 'shown') {
            console.log('Rating dialog displayed');
            clearTimeout(this.ratingTimerId);
        } else if (result === 'dismissed') {
            console.log('Rating dialog dismissed');
        }
    }
}, (err) => {
    console.log('Error opening rating dialog: ' + err);
});
```

**Console output**
<details>
<summary>console output</summary>

```
[Log] Running cordova-ios@6.1.1 (cordova.js, line 1413)
[Log] result: requested (cordova.js, line 1413)
[Log] Requested display of rating dialog (cordova.js, line 1413)
[Warning] Rating dialog was not shown (after 5000ms) (cordova.js, line 1413)
```

</details><br/><br/>

**Other information:**

When the rating dialog is displayed
It has been confirmed that the windowDidBecomeVisibleNotification of LaunchReview.m is called.
At this time, notification.object was not MonitorObject but SKStoreReviewPresentationWindow was returned.

It seems to be related to the issue below
https://github.com/dpa99c/cordova-launch-review/issues/18

**Comment by dpa99c** [comment]
Looks like the underlying iOS implementation for the native in-app review dialog has changed in recent iOS versions which is causing this issue. 
I'll update the code to check for the SKStoreReviewPresentationWindow class name but also leave the check for the MonitorObject class name in order to support older iOS versions.

**Comment by dpa99c** [comment]
Fix has been published in v4.0.1

**Comment by randnetdd** [comment]
Thank you!
